### PR TITLE
updates profile picture with div to wrap to make click more consistent 

### PR DESCRIPTION
**Pull request by AwsomeWyatt**

### CODE REVIEW

File: src/profile-picture-menu.ts [hunk 1]
```diff
@@ -156,7 +156,10 @@ import {ProfilePictureElement} from './profile-picture';
                 </action-buttons>
               </dialog-content>
             </paper-dialog>
-            <profile-picture person-id="[[personId]]" on-click="_onProfilePictureTapped" id="profilePicture" shape="circle" size="[[size]]"></profile-picture>
+            <div>
+				<lss-profile-picture person-id="[[personId]]" on-tap="_onProfilePictureTapped" id="profilePicture" shape="circle"
+				 size="[[size]]"></lss-profile-picture>
+			</div>
```
Comment (aarondrabeck) [inline src/profile-picture-menu.ts]: formatting

File: src/profile-picture-menu.ts [hunk 2]
```diff
@@ -156,7 +156,10 @@ import {ProfilePictureElement} from './profile-picture';
                 </action-buttons>
               </dialog-content>
             </paper-dialog>
-            <profile-picture person-id="[[personId]]" on-click="_onProfilePictureTapped" id="profilePicture" shape="circle" size="[[size]]"></profile-picture>
+            <div>
+				<lss-profile-picture person-id="[[personId]]" on-tap="_onProfilePictureTapped" id="profilePicture" shape="circle"
```
Comment (aarondrabeck) [inline src/profile-picture-menu.ts]: Are you sure that on-tap alone doesn't fix this?

Comment (aarondrabeck) [inline src/profile-picture-menu.ts]: What if you where to give lss-profile-picture person display:block?

File: src/profile-picture-menu.ts [hunk 3]
```diff
@@ -53,110 +53,115 @@ import {ProfilePictureElement} from './profile-picture';
...
+      </conent-container>
+      <action-buttons>
+        <paper-button on-tap="_onLogoutTapped">Logout</paper-button>
+        <slot name="accountButtonSlot">
+          <paper-button main-action on-tap="_onMyAccountTapped" flat>My Account</paper-button>
+        </slot>
+      </action-buttons>
+    </dialog-content>
+  </paper-dialog>
+  <div>
```
[the remark this answers is not in the document]
Comment (AwsomeWyatt) [reply src/profile-picture-menu.ts]: good point

**Pull request merged**